pilot/pkg/bootstrap/server.go

```diff
@@ -834,6 +852,11 @@ func (s *Server) initRegistryEventHandlers() {
 	}
 }
 
+func onlyStatusUpdated(old config.Config, curr config.Config) bool {
+	return labels.Equals(old.Labels, curr.Labels) &&
```

Comment:
does `old.Spec == curr.Spec` actually work? Its a pointer so I would assume not?